source/common/formatter/substitution_formatter.cc

```diff
@@ -816,6 +816,11 @@ StreamInfoFormatter::StreamInfoFormatter(const std::string& field_name) {
         [](const StreamInfo::StreamInfo& stream_info) {
           return stream_info.connectionID().value_or(0);
         });
+  } else if (field_name == "CONNECTION_FINGERPRINT") {
```

Comment:
Maybe the field name should me more specific (e.g. "JA3_TLS_FINGERPRINT") in case there's ever a different kind of fingerprinting added?